package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-plugin-prometheus",
-  "version": "4.2.0",
+  "version": "4.1.1",
```

Comment:
Why did this rollback from `4.2.0` to `4.1.1`?

Reply:
Oops this is my bad